public_src/services/map.service.ts

```diff
@@ -455,8 +459,10 @@ export class MapService {
       this.highlightFill = L.polyline(latlngs, Utils.HIGHLIGHT_FILL).bindTooltip(
         rel.tags.name,
       );
+      this.enableInfoRouteLabelsOption.emit({ type: 'multiple', id: rel.id, highlightFill: this.highlightFill });
```

Comment:
![image](https://user-images.githubusercontent.com/6165660/43365473-be94e260-932d-11e8-84d1-71b696ba93da.png)
can we put here something from the plugin readme so the text is not flipped upside down? If not, we can do it later:

_"orientation Rotate text. (Default: 0)
{orientation: angle} - rotate to a specified angle (e.g. {orientation: 15})
{orientation: flip} - filps the text 180deg correction for upside down text placement on west -> east lines
{orientation: perpendicular} - places text at right angles to the line."_ (https://github.com/makinacorpus/Leaflet.TextPath)

Reply:
I had tried it before, it seems to flip the whole svg left to right, and is no more inside the polylines. But I will try it one more and re confirm.

Reply:
I used orientation: flip for this. This unwanted scenario happens for both directions of the route. Maybe we are supposed to use something else, and there is more details written somewhere, I will try and look.